frontend/wallet/src/render/views/onboarding/InstallCodaStep.re

```diff
@@ -46,31 +44,48 @@ module InstallProgress = {
         <Downloader
           onFinish={result =>
             switch (result) {
-            | Belt.Result.Ok(_) => setState(_ => Finished)
-            | Belt.Result.Error(_) => setState(_ => Finished)
+            | Belt.Result.Ok(_) =>
+              Js.log("Install complete...");
+              Bindings.LocalStorage.setItem(~key=`Installed, ~value="true");
+              setState(_ => Finished);
+            | Belt.Result.Error(err) => setState(_ => Error(err))
             }
           }
           finished={installerState === Finished}
+          error={
+            switch (installerState) {
+            | Error(_) => true
+            | _ => false
+            }
+          }
         />
-        {installerState === Finished
-           ? <p className=Styles.downloaderText>
-               {React.string("Installation Complete!")}
+        {switch (installerState) {
+         | Finished =>
+           <p className=Styles.downloaderText>
+             {React.string("Installation Complete!")}
+           </p>
+         | Error(err) =>
+           <>
+             <p className=Styles.installingText>
+               {React.string("Installation Error")}
              </p>
-           : <>
-               <p className=Styles.installingText>
-                 {React.string("Installing Coda")}
-               </p>
-               <a
-                 onClick={_ =>
-                   openExternal(
-                     "https://codaprotocol.com/docs/troubleshooting",
-                   )
-                 }
-                 target="_blank"
-                 className=Styles.downloaderSubtext>
-                 {React.string({j|Having problems installing Coda?|j})}
-               </a>
-             </>}
+             <p className=Styles.downloaderSubtext> {React.string(err)} </p>
+           </>
+         | _ =>
+           <>
+             <p className=Styles.installingText>
+               {React.string("Installing Coda")}
+             </p>
+             <a
+               onClick={_ =>
+                 openExternal("https://codaprotocol.com/docs/troubleshooting")
```

Comment:
fwiw I don't think this page has much that would help with this right now but that doesn't need to be in this PR